src/lib.rs

```diff
@@ -526,24 +543,33 @@ impl FixedBitSet {
     ///
     /// Iterator element is the index of the `1` bit, type `usize`.
     /// Unlike `ones`, this function consumes the `FixedBitset`.
-    pub fn into_ones(mut self) -> IntoOnes {
-        if self.data.len() == 0 {
+    pub fn into_ones(self) -> IntoOnes {
+        // SAFETY: This is using the exact same allocation pattern, size, and capacity
+        // making this reconstruction of the Vec safe.
+        let mut data = unsafe {
+            let mut data = ManuallyDrop::new(self.data);
+            let ptr = data.as_mut_ptr().cast();
+            let len = data.len() * SimdBlock::USIZE_COUNT;
+            let capacity = data.capacity() * SimdBlock::USIZE_COUNT;
+            Vec::from_raw_parts(ptr, len, capacity)
+        };
```

Comment:
I don't think this is safe. The `Vec` is initially allocated with `SimdBlock`s, which can have a different alignment than `usize`s. For example the one in `avx2` has an alignment of 32 and the one in `sse2` has an alignment of 16, while `usize` only has an alignment of 8.